samplers/jaegerremote/sampler_remote_options.go

```diff
@@ -0,0 +1,130 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Copyright (c) 2021 The Jaeger Authors.
+// Copyright (c) 2017 Uber Technologies, Inc.
+
+package jaegerremote
+
+import (
+	"time"
+
+	"go.opentelemetry.io/otel/sdk/trace"
+)
+
+// SamplerOption is a function that sets some option on the sampler
+type SamplerOption func(options *samplerOptions)
+
+type samplerOptions struct {
```

Comment:
Please align this to our [style guide for configuration](https://github.com/open-telemetry/opentelemetry-go/blob/main/CONTRIBUTING.md#configuration).

Reply:
I always prefer plain config struct over functional options.

Reply:
We have a style guide that we expect instrumentation in this repo to adhere to for consistency.  Preferences of individual authors cannot take precedence.